docs/changes.rst

```diff
@@ -116,6 +116,10 @@ Released: not yet
 * Finalized the support for SI units that was experimental so far, i.e. the
   'pywbem.siunit()' and 'pywbem.siunit_obj()' functions. (issue #2653)
 
+* Modify SubscriptionManager add_filter to use the CIM_IndicationFilter
+  property SourceNamespaces in place of the deprecated SourceNamespace. (see
+  issue #2725)
+
```

Comment:
There should be an additional entry in the Incompatibilities section that points out how this change is incompatible.

Reply:
I know I wrote that.  System locked up yesterday so that must have gotten lost.

DONE